### ISSUE TITLE
Unable to find method getTypeId

### ISSUE THREAD

**Issue by AxlArns** [gvlfm78/BukkitOldCombatMechanics#246]
```
[03:53:01 ERROR]: Could not pass event PlayerQuitEvent to OldCombatMechanics v1.6.6
org.bukkit.event.EventException: null
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:308) ~[spigot-1.13.1.jar:git-Spigot-f6a273b-cd6252a]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[spigot-1.13.1.jar:git-Spigot-f6a273b-cd6252a]
        at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:500) [spigot-1.13.1.jar:git-Spigot-f6a273b-cd6252a]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:485) [spigot-1.13.1.jar:git-Spigot-f6a273b-cd6252a]
        at net.minecraft.server.v1_13_R2.PlayerList.disconnect(PlayerList.java:404) [spigot-1.13.1.jar:git-Spigot-f6a273b-cd6252a]
        at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1356) [spigot-1.13.1.jar:git-Spigot-f6a273b-cd6252a]
        at net.minecraft.server.v1_13_R2.NetworkManager.handleDisconnection(NetworkManager.java:327) [spigot-1.13.1.jar:git-Spigot-f6a273b-cd6252a]
        at net.minecraft.server.v1_13_R2.ServerConnection.c(ServerConnection.java:144) [spigot-1.13.1.jar:git-Spigot-f6a273b-cd6252a]
        at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:998) [spigot-1.13.1.jar:git-Spigot-f6a273b-cd6252a]
        at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:417) [spigot-1.13.1.jar:git-Spigot-f6a273b-cd6252a]
        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:835) [spigot-1.13.1.jar:git-Spigot-f6a273b-cd6252a]
        at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:733) [spigot-1.13.1.jar:git-Spigot-f6a273b-cd6252a]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_181]
Caused by: java.lang.IllegalStateException: Unable to find method getTypeId ([]).
        at com.comphenix.example.NbtFactory.getMethod(NbtFactory.java:356) ~[?:?]
        at com.comphenix.example.NbtFactory.<init>(NbtFactory.java:54) ~[?:?]
        at com.comphenix.example.NbtFactory.get(NbtFactory.java:83) ~[?:?]
        at com.comphenix.example.NbtFactory.getCraftItemStack(NbtFactory.java:207) ~[?:?]
        at com.comphenix.example.Attributes.<init>(Attributes.java:22) ~[?:?]
        at gvlfm78.plugin.OldCombatMechanics.module.ModuleOldArmourStrength.apply(ModuleOldArmourStrength.java:46) ~[?:?]
        at gvlfm78.plugin.OldCombatMechanics.module.ModuleOldArmourStrength.setArmourAccordingly(ModuleOldArmourStrength.java:198) ~[?:?]
        at gvlfm78.plugin.OldCombatMechanics.module.ModuleOldArmourStrength.setArmourToDefault(ModuleOldArmourStrength.java:177) ~[?:?]
        at gvlfm78.plugin.OldCombatMechanics.module.ModuleOldArmourStrength.onPlayerLeave(ModuleOldArmourStrength.java:165) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_181]
        at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_181]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_181]
        at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_181]
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:304) ~[spigot-1.13.1.jar:git-Spigot-f6a273b-cd6252a]
        ... 12 more
```

**Comment by I-Al-Istannen** [comment]
It sounds like a duplicate of #229 which was fixed in PR #230, merged at the 5th of September which I think already is in the latest published OCM version.

If not, please try a dev build from [here](http://ci.rayzr.tech/job/OldCombatMechanics/).

Please report back, if this does not resolve the error for you.

**Comment by gvlfm78** [comment]
@I-Al-Istannen Yeah 1.7 is not released yet because we are still sorting out all issues with #28
